pkg/server/helpers.go

```diff
@@ -405,27 +405,17 @@ func isInCRIMounts(dst string, mounts []*runtime.Mount) bool {
 	return false
 }
 
-//TODO: Replace with `mount.Lookup()`in containerd after #257 is marged
```

Comment:
Why do we still need this loop?
`LookupMount` checks device id directly: https://github.com/containerd/containerd/blob/master/mount/lookup_unix.go